beta-src/src/state/game/game-api-slice.ts

```diff
@@ -128,6 +128,7 @@ export const setVoteStatus = createAsyncThunk(
     vote: string;
     voteOn: string;
   }) => {
+    console.log("Submitting vote");
```

Comment:
I don't think this is meant to be in the PR (and probably should be a lint error)